robottelo/ui/browser.py

```diff
@@ -1,6 +1,15 @@
+"""Tools to help getting a browser instance to run UI tests."""
+import time
+
 from robottelo.config import settings
 from selenium import webdriver
 
+try:
+    import docker
+except ImportError:
+    # Let fail later if not installed
```

Comment:
How do you plan to handle this import failure? Right now things would break when calling `_init_client` method.